app/models/user_response.rb

```diff
@@ -5,16 +5,12 @@ class UserResponse < ActiveRecord::Base
   belongs_to :user
   belongs_to :survey
   belongs_to :company
-  has_many :answers, inverse_of: :user_response, dependent: :destroy
```

Comment:
why remove inverse_of?give description

Reply:
inverse_of saves load time. without inverse_of if we access objects of has_many and belongs_to associations separately then two database hits will be made but with inverse_of if one object is already in memory and we access it through its association then it wont hit database again. Inverse_of has been removed from here because in rails > 4 rails automatically set the inverse_of option so we dont have to mention it explicitly